Snippets/Extensions.DependencyInjection/Extensions.DependencyInjection_1/Usage.cs

```diff
@@ -23,5 +24,16 @@ void ConfigureThirdPartyContainer(EndpointConfiguration endpointConfiguration)
 
             #endregion
         }
+
+        async Task ExternallyManagedMode(EndpointConfiguration endpointConfiguration)
+        {
+            #region externally-managed-mode
+            var serviceCollection = new ServiceCollection();
+
+            var startableEndpoint = EndpointWithExternallyManagedServiceProvider.Create(endpointConfiguration, serviceCollection);
+
+            var endpoint = await startableEndpoint.Start(serviceCollection.BuildServiceProvider());
```

Comment:
should we store the serviceprovider in a dedicated variable and show how it can be used by the user?

Reply:
Sounds reasonable, pushed a proposal

Reply:
The code was not there, so I took a stab at it.

Reply:
Duh, I forgot to push :)